config/data/configs/single.node.config.toml

```diff
@@ -35,7 +35,7 @@ dir = "libradb/db"
 grpc_max_receive_len = 100000000
 
 [validator_network]
-peer_id = "8deeeaed65f0cd7484a9e4e5ac51fbac548f2f71299a05e000156031ca78fb9f"
+peer_id = "b1df0ea1b4c1400454bab824e2e3ef6669e4231e2b9332020d9630fe1cfb2808"
```

Comment:
btw, what are all these configs in data/configs? It's not clear if they are templates (and do we even need templates anymore since you provide defaults now right?)

are they for tests? If so why are they not indicated as tests?

Reply:
Two types: templates and testing. I eliminated the templates since there was an explicit comment in the code that said to rely on code for the default template. The templates in data were actually being compiled into the binary and we still have defaults in several other configs (terraform).